src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,720 @@
+
+        Kill all the jobs belonging to a specif workflow.
+        """
+        logging.info("Going to remove all the jobs for workflow %s", workflow)
+
+        schedd = htcondor.Schedd()
+
+        try:
+            schedd.act(htcondor.JobAction.Remove, "WMAgent_RequestName == %s" % classad.quote(str(workflow)))
+        except RuntimeError:
+            logging.warn("Error while killing jobs on the schedd: WMAgent_RequestName=%s", workflow)
+
+        return
+
+    def updateJobInformation(self, workflow, task, **kwargs):
+        """
+        _updateJobInformation_
+
+        Update job information for all jobs in the workflow and task,
+        the change will take effect if the job is Idle or becomes idle.
+
+        The currently supported changes are only priority for which both the task (taskPriority)
+        and workflow priority (requestPriority) must be provided.
+        """
+        schedd = htcondor.Schedd()
+
+        if 'taskPriority' in kwargs and 'requestPriority' in kwargs:
+            newPriority = int(kwargs['requestPriority']) + int(kwargs['taskPriority'] * self.maxTaskPriority)
+            try:
+                constraint = "WMAgent_SubTaskName =?= %s" % classad.quote(str(task))
+                constraint += " && WMAgent_RequestName =?= %s" % classad.quote(str(workflow))
+                constraint += " && JobPrio =!= %d" % newPriority
+                schedd.edit(constraint, 'JobPrio', classad.Literal(newPriority))
+            except:
+                logging.warn("Failed to update JobPrio for WMAgent_SubTaskName=%s", task)
+
+        return
+
+    #
+    # Start with submit functions
+    #
+    def initSubmit(self, jobList=None):
+        """
+        _makeConfig_
+
+        Make common JDL header
+        """
+        jdl = []
+
+
+        # -- scriptFile & Output/Error/Log filenames shortened to
+        #    avoid condorg submission errors from > 256 character pathnames
+
+        jdl.append("universe = vanilla\n")
+        jdl.append("requirements = %s\n" % self.reqStr)
+
+        jdl.append("should_transfer_files = YES\n")
+        jdl.append("when_to_transfer_output = ON_EXIT\n")
+        jdl.append("log_xml = True\n")
+        jdl.append("notification = NEVER\n")
+        jdl.append("Executable = %s\n" % self.scriptFile)
+        jdl.append("Output = condor.$(Cluster).$(Process).out\n")
+        jdl.append("Error = condor.$(Cluster).$(Process).err\n")
+        jdl.append("Log = condor.$(Cluster).$(Process).log\n")
+
+        jdl.append("+WMAgent_AgentName = \"%s\"\n" % (self.agent))
+        jdl.append("+JOBGLIDEIN_CMSSite= \"$$([ifThenElse(GLIDEIN_CMSSite is undefined, \\\"Unknown\\\", GLIDEIN_CMSSite)])\"\n")
+
+        # Required for global pool accounting
+        jdl.append("+AcctGroup = \"%s\"\n" % (self.acctGroup))
+        jdl.append("+AcctGroupUser = \"%s\"\n" % (self.acctGroupUser))
+        jdl.append("+AccountingGroup = \"%s.%s\"\n" % (self.acctGroup, self.acctGroupUser))
```

Comment:
According to the man page, AcctGroup and AcctGroupUser is correct and AccountingGroup does not exist, so I removed the last line.